test/util.dart

```diff
@@ -107,7 +107,7 @@ class RefTestCaseCollection<T> {
     final calls = [];
     return RefTestCase(
       name: name,
-      ref: (value) => calls.add(value),
+      ref: calls.add,
```

Comment:
Can we retain this version and make it explicitly dynamic, to guard against calls's typing changing?
```suggestion
      // ignore: unnecessary_lambdas, avoid_types_on_closure_parameters
      ref: (dynamic value) => calls.add,
```